docs/superpowers/result.md

```
@@ -0,0 +1,131 @@
+## E2E 失败明细
+
+### FAIL: TestCalendar_CreateEvent
+
+**复现命令：**
+```bash
+LARK_CLI_BIN=/workspace/lark-cli/bin/lark-cli \
+  go test /workspace/lark-cli/tests/cli_e2e/... -run 'TestCalendar_CreateEvent' -v
+```
+
+**输出：**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)